extensions/amp-analytics/0.1/test/test-instrumentation.js

```diff
@@ -106,6 +106,45 @@ describe('instrumentation', function() {
     expect(fnIdY.callCount).to.equal(1);
   });
 
+  it('fires for events on child elements', () => {
+    const el1 = document.createElement('div');
+    const el2 = document.createElement('div');
+
+    const el3 = document.createElement('div');
+    el3.className = 'x z';
+    el3.appendChild(el1);
+
+    const el4 = document.createElement('div');
+    el4.className = 'x';
+    el4.id = 'y';
+    el4.appendChild(el3);
+    el4.appendChild(el2);
+
+    const fnClassX = sandbox.stub();
+    addListener(window, {'on': 'click', 'selector': '.x'}, fnClassX);
```

Comment:
Please rebase. I redid these tests a bit to do `ins.addListener` to improve test isolation - there were a few flakes.